### DISCUSSION TITLE
TPI entrega final

### DISCUSSION THREAD

**Discussion by loyolacamil** [dds-jv/foro#355] [Consultas generales] [answered]
Hola! estoy intentando hacer la entrega final del tpi, pero el mvn verify me sigue mostrando estos dos errores de code smells:

```
[ERROR] Medium: Unread field: ar.edu.utn.frba.dds.macowins.Negocio.ReportesVentas.ReporteDeVentas.fechaVenta [ar.edu.utn.frba.dds.macowins.Negocio.ReportesVentas.ReporteDeVentas] At ReporteDeVentas.java:[line 11] URF_UNREAD_FIELD
[ERROR] Medium: Unread field: ar.edu.utn.frba.dds.macowins.Negocio.Ventas.Venta.fecha [ar.edu.utn.frba.dds.macowins.Negocio.Ventas.Venta] At Venta.java:[line 14] URF_UNREAD_FIELD
```

Ya intente muchos cambios, pero no logro nada.

**Comment by gastonprieto** [comment, marked as answer]
Buenas @loyolacamil 

Buscando `URF_UNREAD_FIELD` llegas a la página de  descripciones de spotbug: https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html ese significa especificamente que no estas usando ese campo, y me parece que la fecha debias usarla para conseguir las ventas en cierto rango?
